### ISSUE TITLE
Dark theme missing

### ISSUE THREAD

**Issue by jahway603** [SecUSo/privacy-friendly-tape-measure#17]
For visual assessibility, a dark theme is missing from this app.

**Comment by DrDustwolf** [comment]
Still the same issue. I installed some of your programms, but as there is no dark/amoled-black/night-mode availlable, I'm not very often using your tools. They're just to bright and hurt my eyes.

I've installed the following of your tools:
Version 2.3.0
org.secuso.privacyfriendlyfinancemanager

Version 1.1.1
org.secuso.privacyfriendlyruler

Version 1.1.1
org.secuso.privacyfriendlytapemeasure

Version 1.2.2
org.secuso.privacyfriendlybattleship

Version 1.3.2
org.secuso.privacyfriendlypaindiary

**Comment by jahway603** [comment]
@DrDustwolf I agree that the apps without a dark theme are to bright and also hurt my eyes.
